Source/WebCore/platform/graphics/FontTaggedSettings.h

```diff
@@ -68,15 +68,15 @@ struct FourCharacterTagHashTraits : HashTraits<FontTag> {
     static constexpr char ff = static_cast<char>(0xFF);
 };
 
-template <typename T>
+template<typename T>
 class FontTaggedSetting {
 private:
     friend struct IPC::ArgumentCoder<FontTaggedSetting, void>;
 public:
     FontTaggedSetting() = delete;
     FontTaggedSetting(FontTag, T value);
 
-    friend bool operator==(const FontTaggedSetting&, const FontTaggedSetting&) = default;
+    bool operator==(const FontTaggedSetting&) const = default;
     bool operator<(const FontTaggedSetting<T>& other) const;
```

Comment:
```suggestion
    bool operator<(const FontTaggedSetting&) const;
```
Also, maybe this should be a `<=>` instead of `<` operator.